packages/flame_3d/lib/src/parser/gltf/animation_interpolation.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:flame_3d/core.dart';
+import 'package:flame_3d/src/parser/gltf/gltf_node.dart';
+
+enum AnimationInterpolation {
+  /// The animated values are linearly interpolated between keyframes.
+  /// When targeting a rotation, spherical linear interpolation (slerp)
+  /// **SHOULD** be used to interpolate quaternions.
+  /// The number of output elements **MUST** equal the number of input elements.
+  linear('LINEAR'),
+
+  /// The animated values remain constant to the output of the first keyframe,
+  /// until the next keyframe.
+  /// The number of output elements **MUST** equal the number of input elements.
+  step('STEP'),
+
+  /// The animation's interpolation is computed using a cubic spline with
+  /// specified tangents.
+  /// The number of output elements **MUST** equal three times the number of
+  /// input elements.
+  /// For each input element, the output stores three elements, an in-tangent,
+  /// a spline vertex, and an out-tangent.
+  /// There **MUST** be at least two keyframes when using this interpolation.
+  cubicSpline('CUBICSPLINE'), // cSpell:ignore CUBICSPLINE
+  ;
```

Comment:
```suggestion
  cubicSpline('CUBICSPLINE'); // cSpell:ignore CUBICSPLINE
```

Reply:
in general I prefer to keep the entries consistent, makes it easier to add more. note that we are changing the GLTF spec anytime soon, but I'd rather follow convention for all enums